### ISSUE TITLE
Task: MIRI - LRS Slitless

### ISSUE THREAD

**Issue by brechmos-stsci** [spacetelescope/jwst#267]
Things to do
- [x]  Implement algorithm for spatial coordinates

**Comment by brechmos-stsci** [comment]
**Blocked** 
- We are waiting for an algorithm for spatial coordinates (waiting on INS)

**Comment by hbushouse** [comment]
While we definitely want to have spatial coords assigned eventually, this mode technically doesn't need them, because these data will always be used in time-series mode observations, which means we won't ever be combining or resampling the data, hence there's no need to know the spatial coords for any downstream steps. All we need are the wavelength assignments per pixel, which we already have, in order to perform 1-D spectral extraction from the 2-D images (and produce a wavelength vector in the process).

**Comment by nden** [comment]
After talking with Howard, we will implement the following:

    identify the row (cross-dispersion direction) of the zero point
    compute V2, V3 (RA, DEC) for each pixel in this row
    replicate this solution for all rows in the fixed slit and slitless domain

I am removing the blocked label but still would like to hear what the INS team thinks should be done.

**Comment by brechmos-stsci** [comment]
@nden  @hbushouse suggested in https://github.com/STScI-JWST/jwst/pull/347#discussion_r83409866 that we should be pulling the rotation angle correction (currently hardcoded as 0.00421924) from configuration file. There seem to be a few options. 

Should it go in the MIRI distortion file (jwst_miri_distortion_0001.asdf)?  If so, who do we contact to make sure it is added officially?  I can add it to mine locally, for now, to the appropriate configuration file.

**Comment by brechmos-stsci** [comment]
@nden and I looked over the code, data and correction files yesterday.  

**Slitless Prism Data**
- File is jw00025001001_01107_00002_MIRIMAGE_rate.fits
- Image is 416 x 72
- Sub-array xstart = 1, ystart = 529

**Spec Correction File**
- File is MIRI_FM_MIRIMAGE_P750L_DISTORTION_06.02.00.fits
- table that is 388 x 11
- zero point is (34, 763)

Based on these, the domain is calculated as:
```
[
    {'lower': 12.772724032479712, 'upper': 54.405568289975079}, 
    {'lower': 471.90038528245242, 'upper': 859.19741287034299}
]
```

[I'll add more to this if it is helpful.]

**Comment by nden** [comment]
Resolved by #3106 and others before that.
